opentok/opentok.py

```diff
@@ -1365,3 +1367,66 @@ def __init__(
             timeout=timeout,
             app_version=app_version
         )
+
+    
+
+
+    def mute(self, session_id: str, stream_id: str= "", options: dict = {}) -> requests.Response:
+        """
+        Use this method so the moderator can mute all streams or a specific stream for OpenTok.
+        Please note that a client is able to unmute themselves.
+        This function stays in the OpenTok class and inherits from the Client class.
+
+        :param session_id gets the session id from another function called get_session()
+
+        :param stream_id gets the stream id from another function called get_stream(). Note
+        that this variable is set to an empty string in the function definition as a specific
+        stream may not be chosen.
+   
+        """
+
+        session_id = self.get_session()
```

Comment:
We won't need to regenerate a session, we can use the `session_id` that is passed in above

Reply:
Hi @dragonmantank I made this change and pushed my code yesterday. I wasn't sure how to re-tag you in the pull request, so I'm leaving a comment instead :-)